udata_front/theme/gouvfr/datagouv-components/src/helpers/i18n.ts

```diff
@@ -1,11 +1,10 @@
 import dayjs from "dayjs/esm";
 import LocalizedFormat from 'dayjs/esm/plugin/localizedFormat';
 import RelativeTime from "dayjs/esm/plugin/relativeTime";
-import { DefineLocaleMessage, I18n, Locale, createI18n } from "vue-i18n";
+import { type I18n, createI18n } from "vue-i18n";
 import messages from '@intlify/unplugin-vue-i18n/messages';
 import { config } from "../config";
 import { api_root_absolute } from "../api/api";
-import { getRegisteredTranslations } from "@etalab/udata-front-plugins-helper";
```

Comment:
everything related to udata-front-plugins-helper is unused so it's removed.

removing udata-front-plugins-helper is the next step but it significant enough to be a separate PR